sdk/maps/azure-maps-render/src/main/java/com/azure/maps/render/RenderAsyncClient.java

```diff
@@ -4,45 +4,48 @@
 
 package com.azure.maps.render;
 
+import java.nio.ByteBuffer;
+import java.util.Arrays;
+import java.util.List;
+
 import com.azure.core.annotation.ReturnType;
 import com.azure.core.annotation.ServiceClient;
 import com.azure.core.annotation.ServiceMethod;
 import com.azure.core.http.rest.Response;
 import com.azure.core.http.rest.StreamResponse;
+import com.azure.core.util.Context;
 import com.azure.maps.render.implementation.RenderClientImpl;
+import com.azure.maps.render.implementation.RenderV2sImpl;
+import com.azure.maps.render.implementation.helpers.Utility;
+import com.azure.maps.render.implementation.models.BoundingBoxPrivate;
+import com.azure.maps.render.implementation.models.Copyright;
+import com.azure.maps.render.implementation.models.CopyrightCaption;
+import com.azure.maps.render.implementation.models.ErrorResponseException;
+import com.azure.maps.render.implementation.models.MapAttribution;
+import com.azure.maps.render.implementation.models.MapTilesetPrivate;
+import com.azure.maps.render.implementation.models.ResponseFormat;
+import com.azure.maps.render.implementation.models.TileIndex;
+import com.azure.maps.render.implementation.models.TilesetID;
 import com.azure.maps.render.models.BoundingBox;
-import com.azure.maps.render.models.Copyright;
-import com.azure.maps.render.models.CopyrightCaption;
-import com.azure.maps.render.models.ErrorResponseException;
-import com.azure.maps.render.models.IncludeText;
-import com.azure.maps.render.models.LocalizedMapView;
-import com.azure.maps.render.models.MapAttribution;
-import com.azure.maps.render.models.MapImageStyle;
-import com.azure.maps.render.models.MapTileSize;
-import com.azure.maps.render.models.MapTileset;
-import com.azure.maps.render.models.RasterTileFormat;
-import com.azure.maps.render.models.ResponseFormat;
-import com.azure.maps.render.models.StaticMapLayer;
-import com.azure.maps.render.models.TileIndex;
-import com.azure.maps.render.models.TilesetID;
-import java.nio.ByteBuffer;
-import java.time.OffsetDateTime;
-import java.util.List;
+import com.azure.maps.render.models.LatLong;
+import com.azure.maps.render.models.MapStaticImageOptions;
+import com.azure.maps.render.models.MapTileOptions;
+
 import reactor.core.publisher.Flux;
 import reactor.core.publisher.Mono;
 
 /** Initializes a new instance of the asynchronous RenderClient type. */
 @ServiceClient(builder = RenderClientBuilder.class, isAsync = true)
 public final class RenderAsyncClient {
-    private final RenderClientImpl serviceClient;
+    private final RenderV2sImpl serviceClient;
 
     /**
      * Initializes an instance of RenderClient client.
      *
      * @param serviceClient the service client implementation.
      */
     RenderAsyncClient(RenderClientImpl serviceClient) {
```

Comment:
It's probably better to pass the RenderV2 object directly. This way when mocking you don't need to construct a whole RenderClientImpl!